packages/kit/src/cli.js

```diff
@@ -200,16 +200,21 @@ prog.parse(process.argv, { unknown: (arg) => `Unknown option: ${arg}` });
 
 /** @param {number} port */
 async function check_port(port) {
-	const n = await ports.blame(port);
-
-	if (n) {
+	if (!(await ports.check(port))) {
```

Comment:
you could reverse this check so that there's less code indentation and you don't have everything wrapped in a big `if`:
```
if (await ports.check(port)) {
  return;
}
```